selfdrive/controls/controlsd.py

```diff
@@ -230,11 +230,12 @@ def update_events(self, CS):
     if self.sm['plan'].fcw:
       self.events.add(EventName.fcw)
     if self.sm['model'].frameDropPerc > 1:
-      self.events.add(EventName.modeldLagging)
+      #self.events.add(EventName.modeldLagging)
```

Comment:
This needs to be gated between an environment variable. Maybe `SIMULATION=1`

Reply:
hmm, this should run fast enough with the keras runner